src/main/java/org/example/Extractor.java

```diff
@@ -3,23 +3,50 @@
 import org.jsoup.Jsoup;
 import org.jsoup.nodes.Document;
 import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
 public class Extractor {
     public List<Chair> getChairs(String url){
-
-        String role, name, email;
         List<Chair> chairs = new ArrayList<>();
-        //todo
+        try {
+            Document doc = Jsoup.connect(url).get();
+            Elements people = doc.select("tr");
+            String role, name, email;
+            for (Element individual : people) {
+                String title = individual.select("td").text();
+                boolean chair = title.toLowerCase().contains("chair"); // Check if their affiliation contains the word "Chair" or not.
+                if (chair == true) {
```

Comment:
Placing this inside the if statement straight away instead of storing it in a variable might be better.

Reply:
This would make the code more readable.

Reply:
I actually meant something else, but your latest implementation is better than what I had in mind.